tests/B2CWebAppCallsWebApi/Client/Startup.cs

```diff
@@ -53,9 +53,7 @@ public void ConfigureServices(IServiceCollection services)
 
             // Token acquisition service based on MSAL.NET
             // and chosen token cache implementation
-            services.AddWebAppCallsProtectedWebApi(Configuration,
-                initialScopes: new string[] { Configuration["TodoList:TodoListScope"] },
-                configSectionName: "AzureAdB2C")
+            services.AddWebAppCallsProtectedWebApi(Configuration, "AzureAdB2C")
```

Comment:
Just wanted to point out the difference between AAD and B2C: 

- AAD will return an access token if you pass 0 scopes. AcquireTokenSilent works.
- B2C will not return an access token if you pass 0 scopes. This is a known issue that B2C team plans to fix at some point. AcquireTokenSilent does not work (MSAL will thrown an exception). AcquireToken*** will still return an ID token though.

So, long story short, I'm not sure you want to make this change for B2C ?

Reply:
yes. i just talked about it w/ @jmprieur .I think we will document it for now. I'll let @jmprieur make that decision.